policyengine_us/variables/gov/states/hi/tax/income/credits/cdcc/hi_cdcc.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class hi_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii child and dependent care credit"
+    defined_for = StateCode.HI
+    unit = USD
+    definition_period = YEAR
+    reference = "https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=40"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.credits.cdcc
+        agi = tax_unit("adjusted_gross_income", period)
```

Comment:
```suggestion
        agi = tax_unit("hi_agi", period)
```

Reply:
need to create empty variable for which we will create a formula later (Form N-11, line 20)